File: beecrowd/go/bee1099.go
package main

import "fmt"

func main() {

	var num, sum, in1, in2 int

	fmt.Scanf("%d", &num)

	for i := 0; i < num; i++ {
		sum = 0

		fmt.Scanf("%d %d", &in1, &in2)

		if in2 < in1 {
			var temp int = in1
			in1 = in2
			in2 = temp
		}

		for in1 = in1 + 1; in1 < in2; in1++ {

			if in1%2 != 0 {
				sum += in1
			}
		}

		fmt.Printf("%d\n", sum)
	}

}

File: beecrowd/c/bee1099.c
#include <stdio.h>

int main() {
    int n, x, y, sum;
    n = x = y = 0;

    scanf("%d", &n);

    for (int i = 0;i < n; i++) {
        sum = 0;

        scanf("%d", &x);
        scanf("%d", &y);

        if (y < x) {
          int temp = x;
          x = y;
          y = temp;
        }

        for (x += 1;x < y; x++) 
          if (x % 2 != 0) sum += x;
        
        printf("%d\n", sum);
    }
    
    return 0;
}
